[2024-11-29T00:00:47.199+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-29T00:00:47.209+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.download_dataset_task scheduled__2024-11-28T00:00:00+00:00 [queued]>
[2024-11-29T00:00:47.213+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.download_dataset_task scheduled__2024-11-28T00:00:00+00:00 [queued]>
[2024-11-29T00:00:47.213+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-11-29T00:00:47.218+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): download_dataset_task> on 2024-11-28 00:00:00+00:00
[2024-11-29T00:00:47.224+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=54985) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-29T00:00:47.225+0000] {standard_task_runner.py:72} INFO - Started process 54986 to run task
[2024-11-29T00:00:47.226+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'download_dataset_task', 'scheduled__2024-11-28T00:00:00+00:00', '--job-id', '109', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmp6yudj7nz']
[2024-11-29T00:00:47.228+0000] {standard_task_runner.py:105} INFO - Job 109: Subtask download_dataset_task
[2024-11-29T00:00:47.326+0000] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_gcs_dag.download_dataset_task scheduled__2024-11-28T00:00:00+00:00 [running]> on host 4cef807186dc
[2024-11-29T00:00:47.427+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_gcs_dag' AIRFLOW_CTX_TASK_ID='download_dataset_task' AIRFLOW_CTX_EXECUTION_DATE='2024-11-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-28T00:00:00+00:00'
[2024-11-29T00:00:47.427+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-29T00:00:47.435+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-11-29T00:00:47.436+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'curl -sSL https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2021-01.csv > /opt/***/yellow_tripdata_2021-01.csv']
[2024-11-29T00:00:47.444+0000] {subprocess.py:99} INFO - Output:
[2024-11-29T00:00:47.697+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2024-11-29T00:00:47.712+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-29T00:00:47.713+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=download_dataset_task, run_id=scheduled__2024-11-28T00:00:00+00:00, execution_date=20241128T000000, start_date=20241129T000047, end_date=20241129T000047
[2024-11-29T00:00:47.743+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-29T00:00:47.758+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-29T00:00:47.759+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
